### DISCUSSION TITLE
Docker window image with Private Nuget package reference

### DISCUSSION THREAD

**Discussion by atulthummar** [NuGet/Home#11261] [Q&A]
Hi Team,

Can you please provide input on Docker window image with Private Nuget package reference Or best approach to achieve this feature.

Thanks

**Comment by atulthummar** [comment]
Used Azure DevOps Artifact and create Private Access Token to access

**Reply by erdembayar** [reply]
I'm glad you solved your problem.
It would be great if you post sample docker image  here, it'll help for next time as reference. Please replace any actual token with placeholder if have any in your image.
